pkg/urlutil/git_test.go

```diff
@@ -9,6 +9,7 @@ var (
 		"git@bitbucket.org:atlassianlabs/atlassian-docker.git",
 		"https://github.com/docker/docker.git",
 		"http://github.com/docker/docker.git",
+		"http://github.com/docker/docker.git#branch",
```

Comment:
I think you support `.../docker.git#:dir` - meaning 'master' branch - so I think we should test it and add it to the table.